src/List/ListItem.js

```diff
@@ -38,6 +39,16 @@ export const styleSheet = createStyleSheet('ListItem', (theme) => {
       paddingLeft: 16,
       paddingRight: 16,
     },
+    button: {
```

Comment:
Well, tried that `Button` in `ListItem`, but it seems its not ready yet, styles and paddings were weird.